go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/zeta-chain/zetacore
 
-go 1.22
+go 1.22.2
```

Comment:
This is happening because of https://github.com/cometbft/cometbft-db/blob/76f098c698097f0dc9742e06ae964c29440f0f28/go.mod#L3